docs/components/demo/TimeRangeField/css/styles.css

```
@@ -0,0 +1,56 @@
+  line-height: 1.25rem;
+  color: var(--gray-9);
+}
+
+.TimeField {
+  display: flex;
+  padding: 0.5rem;
+  align-items: center;
+  border-radius: 0.25rem;
+  border-width: 1px;
+  text-align: center;
+  background-color: #ffffff;
```

Stylelint (17.2.0)

[error] Expected "#ffffff" to be "#fff" (color-hex-length)

(color-hex-length)